qcfractal/snowflake.py

```diff
@@ -315,7 +315,6 @@ def start(self, timeout: int = 5) -> None:
         self._qcfractal_proc = _background_process([
             shutil.which("qcfractal-server"),
             "start",
-            f"--database-name={self._dbname}",
```

Comment:
This will create some restart issues. I think we will need to init a new temporary folder and `qcfractal-init` into that every time.